src/agent/agent.h

```diff
@@ -79,6 +87,7 @@ bool agent_parse_config(Agent *agent, const char *configfile);
 
 bool agent_start(Agent *agent);
 bool agent_stop(Agent *agent);
+bool agent_connect(Agent *agent);
```

Comment:
Who do you envision calling this method? To me it seems local to the agent

Reply:
```agent_connect()``` was originally only going to be called in agent.c, and due to your comment, I was going to make this function static.  However, it was suggested this also be called from main.c.